### ISSUE TITLE
Should use "roaming" instead of "local" user settings folder

### ISSUE THREAD

**Issue by fritzophrenic** [fireout/keepasssequencer#27]
As discussed in #24 the "correct" folder for configuration data is probably "roaming" instead of "local".

This will allow people who log into a Windows Domain to have their settings migrate with them from computer to computer.

Or something like that.

Make sure the chosen solution works on Linux as well (via Mono) or it's a nonstarter.

**Comment by fireout** [comment]
I fixed this in https://github.com/fireout/keepasssequencer/commit/1ac14c13a7f9a7c59f92cbaa99ad12083c623c3b but I'm not sure how to test this with Linux.

I'm more of a CLI linux user, I'll try to test under Feroda when I'll know where those folders are...  @fritzophrenic: Can you test this in your Linux setup?

**Comment by fritzophrenic** [comment]
<s>See my comments on the commit, it crashes on my Windows 7 system.</s> <sub>Updated: it's not actually related to this change.</sub>

But it seems to work fine on my Windows Vista system, and my Ubuntu system at home.

It pops up the "missing or corrupt" error message the first time it's launched, that is not very user-friendly. Probably it would be better to silently create the file if it just doesn't exist.

**Comment by fritzophrenic** [comment]
I should mention: on Ubuntu it places files in ~/.config. Previously it went to ~/.local/share.

**Comment by fritzophrenic** [comment]
As indicated, it seems to work fine on all systems now. I ran into a bug noted in the comments I made on the commit initially.

The transition is not very user-friendly; the first time the user launches the config dialog they will get a "file missing or corrupt" error message because it is now looking in a new directory. Can we do something about that? Maybe automatically copy the %LOCALAPPDATA% config into %APPDATA% if it is found?
